content/articles/image-adjustment-to-higher-or-lower-resolution-using-python/index.md

```diff
@@ -0,0 +1,238 @@
+
+> This is different from scaling up an original image.
+
+```py
+import cv2
+import numpy as np
+img = cv2.imread("sean.jpg")
+#We will be factoring down images using the already scaled.
+lwr1 = cv2.pyrDown(img)
+lwr2 = cv2.pyrDown(lwr1)
+lwr3 = cv2.pyrDown(lwr2)
+lwr4 = cv2.pyrDown(lwr3)
+# We will be Increasing the resolution of already scaled down image that is lwr4.
+hir1 = cv2.pyrUp(lwr3)
+
+cv2.imshow("Original image", img)
+cv2.imshow("First Scaled Down Image", lwr1)
+cv2.imshow("Second Scaled Down Image", lwr2)
+cv2.imshow("Third Scaled Down Image", lwr3)
+cv2.imshow("Fourth Scaled Down Image", lwr4)
+cv2.imshow("First Scaled Up Image", hir1)
+
+cv2.waitKey(0)
+cv2.destroyAllWindows()
+```
+
+The output should be similar to the below image:
+
+![high](/engineering-education/image-adjustment-to-higher-or-lower-resolution-using-python/high.jpg)
+
+### Instance 3
+To construct the Laplacian Pyramid Super-Resolution Network, we will be taking the top-level layer of the Gaussian Pyramid which is the last image generated using the loop function then create a list for the Laplacian pyramid then create the extended version of the upper level of the Gaussian Pyramid.
+To understand it better a code snippet will do:
+
+```py
+import cv2
+img = cv2.imread("sean.jpg")
+layer = img.copy()
+gp = [layer]
+for j in range(2):
+    layer = cv2.pyrDown(layer)
+    gp.append(layer)
+
+layer = gp[2]
+cv2.imshow("Gausian Upper level", layer)
+lp = [layer]#Introduce and create a list for the Laplacian Pyramid.
+
+for j in range(2, 0, -1):
+    Gausian_extended = cv2.pyrUp(gp[j])#Creating the Laplacian Pyramid.
+laplacian = cv2.subtract(gp[j-1], Gausian_extended)
+cv2.imshow(str(j), laplacian)
+
+cv2.imshow("Original image", img)
+cv2.waitKey(0)
+cv2.destroyAllWindows()
+```
+
+Confirm with the below displayed:
+
+![lapsn](/engineering-education/image-adjustment-to-higher-or-lower-resolution-using-python/lapsn.png)
+
+The Laplacian pyramid and the Gaussian Pyramid are applied in the blending and reconstruction of the images.
+
+### Application fields for adjusting image resolution
+- **Applied in the field of Astronomy**: This is achieved by focusing on tiny information of images to higher resolutions that bring out clearer vision.
+- **Closed Circuit Television**: Feeds received from the CCTV footage can be focused to get clearer images by resolving and fixing the colors where possible.
+- **Medicine**: Images received from scans such as the X-Rays require adjustment of resolutions to provide detailed and accurate information.
+
+### Conclusion
+Covering all the details required in this tutorial, you are now equipped to handle matters concerning this tutorial. Being able to accurately work with this area of discussion and produce maximum results now, wish you all the best.
```

Comment:
I feel this conclusion is very basic. Give a better conclusion that summarizes the content of your article briefly.